### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by clcoffey**
I added the instruction to specify Why a logo is being added to the website.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/add--org--logo-to-the-website.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ You can delete the whole `### Title` section from here.
 - [ ] please edit the InnerSource in Action page located at the following path: content/community/action.md
 - [ ] please add your logo in the correct place in the list, following alphabetical order, by using the following code:
 {{< company name="organisation_name" image="/images/logos/organisation_name.png" >}}{{< /company >}}
-
+- [ ] in the Pull Request please specify why you are adding the logo to the site (E.g.: I work for the company and know they are doing InnerSource)
```
Comment (spier) [inline .github/ISSUE_TEMPLATE/add--org--logo-to-the-website.md]:
```suggestion
- [ ] please specify in the Pull Request why you are adding the logo to the site (E.g.: I work for the company and know they are doing InnerSource)

```

Comment (spier) [reply .github/ISSUE_TEMPLATE/add--org--logo-to-the-website.md]: Matching sentence structure to previous bullets.
Adding spacing ahead of headline.

**Review by spier — APPROVED** [review]
Suggested one wording fix.

One question:
Why are these multiple dashes in the filename `add--org--logo-to-the-website.md`?
Is this some sort of requirement of GH issue templates?

**Comment by clcoffey** [comment]
I am not sure. The file is created by GitHub, I just created the templated through their interface.

**Pull request merged**